paasta_tools/utils.py

```diff
@@ -81,6 +81,8 @@
 
 INSTANCE_TYPES = ('marathon', 'chronos', 'paasta_native', 'adhoc')
 
+DOCKER_ARGS_WHITELIST = ('shm-size')
```

Comment:
why do we really need a whitelist?

Is the attack vector someone editing their config using something like --privileged? (which is another requested argument)

I don't think we should be trying to protect against things like that at this layer in the codebase. If we are going to allow free-form docker arguments, just make it free-form.